cmd/util/cmd/find-inconsistent-result/cmd.go

```diff
@@ -189,11 +198,11 @@ func findOwnResultIDByBlockID(results storage.ExecutionResults, blockID flow.Ide
 	return result.ID(), nil
 }
 
-func findSealedResultIDByBlockHeight(state protocol.State, blockID flow.Identifier) (flow.Identifier, error) {
-	result, _, err := state.AtBlockID(blockID).SealedResult()
```

Comment:
this was wrong, because it's getting the result of latest seal included in this block, which is different from the seal of the given block